### DISCUSSION TITLE
Docker - zigbee2mqtt

### DISCUSSION THREAD

**Discussion by comgenKiffKeff** [Koenkk/zigbee2mqtt#18584] [General]
Hello together, 

I want to move my SmartHome to my Docker environment (via Portainer). 

But somehow I can't get zigbee2mqtt to work. I am having trouble with the volumes, because I want the config file to not be inside the docker environment.

Can anyone help me with this?

**Comment by DubhAd** [comment]
Did you follow the [Docker documentation](https://www.zigbee2mqtt.io/guide/installation/02_docker.html)?
